From 5cc0caeeb63408fab0005fb6dd77237ca00fb570 Mon Sep 17 00:00:00 2001
From: maxice8 <thinkabit.ukim@gmail.com>
Date: Wed, 23 Jan 2019 01:51:29 -0200
Subject: [PATCH] add missing stddef.h include for 'NULL'

clientid.c: In function 'mailimap_clientid':
clientid.c:66:38: error: 'NULL' undeclared (first use in this function)
   if (mailimap_read_line(session) == NULL)
                                      ^~~~
clientid.c:66:38: note: 'NULL' is defined in header '<stddef.h>'; did you forget to '#include <stddef.h>'?
clientid.c:39:1:
+#include <stddef.h>

clientid.c:66:38:
   if (mailimap_read_line(session) == NULL)
---
 src/low-level/imap/clientid.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/low-level/imap/clientid.c b/src/low-level/imap/clientid.c
index 1c34637..f28b616 100644
--- src/low-level/imap/clientid.c
+++ src/low-level/imap/clientid.c
@@ -33,6 +33,8 @@
 #	include <config.h>
 #endif
 
+#include <stddef.h>
+
 #include "mailimap_sender.h"
 #include "clientid_sender.h"
 #include "clientid.h"
-- 
2.20.1


